doc/reference/misc.md

```diff
@@ -402,6 +402,52 @@ within the body.
 :::
 
 
+## Bytes
+::: tip To use the bindings from this module:
+``` scheme
+(import :std/misc/bytes)
+```
+:::
+
+### bytevector-swap!
```

Comment:
let's document the `u8vector-` versions as well, document one and declare the other on as an alias.

Reply:
Do you want examples for each or a simple description of the alias is fine?

Reply:
I simply added alias definitions. I do not think other examples add any value.

Reply:
yeah, that's fine.